portal/src/app/components/home/components/experiment/components/table-options/TableOptions.tsx

```diff
@@ -0,0 +1,34 @@
+import styles from './TableOptions.module.scss';
+import { TABLE_OPTIONS_EN } from './translate/en';
+import SelectColumnsSvg from '../../../../../../../assets/images/select-columns.svg';
+import SelectColumnsCheckedSvg from '../../../../../../../assets/images/select-columns-checked.svg';
+import { Button, ButtonActionType, ButtonSize, ButtonStyleType } from '../../../../../../shared/components/att-button';
+
+const SelectColumnsAriaLabel: string = TABLE_OPTIONS_EN.SELECT_COLUMNS;
+
+export interface TableOptionsProps {
+  handleSelectColumnsClick: () => void;
+  isPopupOpen: boolean;
+  // data: AttSelectOption[];
+}
+
+export const TableOptions: React.FC<TableOptionsProps> = (props: TableOptionsProps) => {
+  return (
+    <div className={styles.table_options_wrapper}>
+      <Button
+        className={styles.close_button}
```

Comment:
remove `className` not in use
please also check that all the className are in use